src/components/ArticlePage.jsx

```diff
@@ -1,33 +1,58 @@
 import { useState, useEffect } from 'react'
 import { useParams } from 'react-router-dom';
-import { getArticles } from '../axios';
+import { getArticles, getComments } from '../axios';
+import CommentCard from './CommentCard'
 
 const ArticlePage = () => {
 
     const { article_id } = useParams();
-    const [currentArticle, setCurrentArticle] = useState({})
+    const [currentArticle, setCurrentArticle] = useState({});
     const [isLoading, setIsLoading] = useState(true);
 
+    const [commentList, setCommentList] = useState([]);
+    const [showComments, setshowComments] = useState(false);
+
+
     useEffect(() => {
         console.log("article-effect activated")
```

Comment:
Generally, we want to make sure that we get rid of Console.Logs we dont really need :)